src/applications/auth/containers/AuthApp.jsx

```diff
@@ -132,8 +133,13 @@ export class AuthApp extends React.Component {
     const returnUrl = sessionStorage.getItem(authnSettings.RETURN_URL) || '';
     sessionStorage.removeItem(authnSettings.RETURN_URL);
 
+    const postAuthUrl =
+      returnUrl.includes('?next=') && !environment.isProduction()
+        ? returnUrl.concat('&postLogin=true')
```

Comment:
Might be a good place to use `URLSearchParams`.

Reply:
Ah, I keep forgetting we don't have `URL` polyfilled since we're still on `core-js` v2... Well, at the least, we can maybe use `append-query` here.

Reply:
That's a good point. Just updated with `append-query`.